vdev/src/commands/meta/mod.rs

```diff
@@ -1,5 +1,6 @@
 crate::cli_subcommands! {
     "Collection of meta-utilities..."
+    mod compile_vrl_wasm,
```

Comment:
I'm not sure this qualifies as a `meta` command. I think you can put this at the top level for now, I intend to rearrange a couple of things to fit.

Reply:
I will move this to the top level now